Expensify/App
# Make cash have all betas enabled when running in development mode

### Details


### Fixed Issues

Fixes https://expensify.slack.com/archives/C03TQ48KC/p1621974182298000

### Tests
- Make the api User_GetBetas return `[]`
- Load the app before this change, make sure request money option is not shown in the big plus button
- Load the app after this change, make sure request money option is shown in the big plus button

### QA Steps
None, since this is just for dev

### Tested On
